lib/api/tenure/v1/types.ts

```diff
@@ -0,0 +1,116 @@
+export interface TenureDetailsProperties {
+  title: string;
+}
+
+// Once other possible param values are confirmed, we can reuse this for TenureType code
+export enum TenureTypeCode {
+  INT = 'INT',
+}
+
+export interface TenureType {
+  code: string;
+  description: string;
+}
+
+export enum TenureAssetType {
+  DWELLING = 'dwelling',
+  GARAGE = 'garage',
+}
+
+export type PersonTenureType =
+  | 'Tenant'
+  | 'Leaseholder'
+  | 'Freeholder'
+  | 'HouseholdMember';
+
+export interface HouseholdMember {
+  id: string;
+  type: string;
+  fullName: string;
+  isResponsible: boolean;
+  dateOfBirth: string;
+  personTenureType?: PersonTenureType;
+}
+
+export interface TenureAsset {
+  id: string;
+  type: string;
+  fullAddress: string;
+  uprn: string;
+}
+
+export interface AccountType {
+  code: string;
+  description: string;
+}
+
+export interface Charges {
+  rent: number;
+  currentBalance: number;
+  billingFrequency: string;
+  paymentReference: string;
+  rentGroupCode: string;
+  rentGroupDescription: string;
+  serviceCharge: number;
+  otherCharges: number;
+  combinedServiceCharges: number;
+  combinedRentCharges: number;
+  tenancyInsuranceCharge: number;
+  originalRentCharge: number;
+  originalServiceCharge: number;
+}
+
+export interface AgreementType {
+  code: string;
+  description: string;
+}
+
+export interface NoticeType {
+  type: string;
+  servedDate: string;
+  expiryDate: string;
+  endDate: string;
+  effectiveDate: string;
+}
+
+export interface LegacyReferences {
+  name: string;
+  value: string;
+}
+
+export interface TenureSectionProps {
```

Comment:
Not sure if splitting this here makes sense under this context. `TenureSectionProps` is used by a component I believe.